ha/core/system_health/system_health.py

```diff
@@ -444,6 +442,12 @@ def process_event(self, healthevent: HealthEvent):
                 # healthevent is functional_type
                 if specific_info and specific_info.get('functional_type'):
                    healthevent.specific_info['functional_type'] = specific_info.get('functional_type')
+
+            status = self.statusmapper.map_event(healthevent)
```

Comment:
status needs to be populated even when current_health is NULL
That needs to be addressed if map_event is moved here

Reply:
I just have moved this statement because there was a need. And if you check, it will be logged irrespective of whether there will be current health or not.

Reply:
Got It. It is outside the if check only